app/models/work.rb

```diff
@@ -8,6 +8,9 @@ class Work < ActiveFedora::Base
   # Change this to restrict which works can be added as a child.
   # self.valid_child_concerns = []
   validates :title, presence: { message: 'Your work must have a title.' }
+  property :genre, predicate: 'http://purl.org/dc/elements/1.1/type' do |index|
```

Comment:
Can we use `RDF::Vocab::DC11.type` instead of the URI string?

Using the `rdf-vocab` gem reduces the chance of minor errors and typos in common vocabulary terms. It also should reduce total object (`String`, `RDF::URI`) allocations (though probably not enough to matter).

Reply:
@no-reply I'm totally cool with that change in the longer term, but this PR came out of a session we were all doing this morning where we were really trying to emphasize the parallels between the metadata spreadsheet and what goes in our code. Seems like working with RDF is a little new for folks, and I'd like to get us there step by step. Can we put in a separate ticket for revising our metadata to use `RDF::Vocab` after everyone has their feet under them?

Reply:
That works for me!

Maybe let's plan a longer RDF.rb session in a couple of weeks. I bet most of the DCE team would like to sit in on that, too.

Reply:
Yes please! :+1: